buildlib/azure-pipelines.yml

```diff
@@ -134,3 +134,16 @@ stages:
               ./contrib/configure-release
               ./contrib/buildrpm.sh -s -t -b
             displayName: Build tarball
+
+  - stage: Tests
+    dependsOn: [Codestyle]
+    jobs:
+    - template: tests.yml
+      parameters:
+        name: althca
+        demands: althca -equals yes
+    # - template: tests.yml
+    #   parameters:
+    #     name: gpu
+    #     demands: gpu -equals yes
```

Comment:
remove?

Reply:
Yes, that was to give an idea on how to add other configurations. Mostly for myself.